assets/queries/cloudFormation/filter_and_alarm_exist_for_for_changes_to_network_gateways/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_S3_Changes_To_Network_Gateways",
+  "queryName": "Filter And Alarm Not Exists For Changes To Network Gateways",
```

Comment:
Change **directory name** and **id** to: `network_gateway_changes_without_alarms`
Change **query name** to: `Network Gateway Changes Without Alarms`